doc/api/assert.markdown

```diff
@@ -2,8 +2,9 @@
 
     Stability: 2 - Stable
 
-This module is used for writing assertion tests. You can access it with
-`require('assert')`.
+This module is used so that Node.js can test itself. You can access it with
+`require('assert')`. However, it is recommended that you use a userland
+assertion library instead.
```

Comment:
I'm not actually sure I agree with this, since it already exists.

Might as well just deprecate it then, no?